helm/ph-ee-engine/templates/tests/integration-test.yaml

```diff
@@ -46,6 +44,20 @@ spec:
               value: "{{ .Values.integration_test.loan.base_url }}"
             - name: "SAVINGS_BASE-URL"
               value: "{{ .Values.integration_test.savings.base_url }}"
+            - name: CHANNEL_CONTACTPOINT
+              value: "{{ .Values.integration_test.channel.contactpoint }}"
+            - name: SAVINGS_CONTACTPOINT
+              value: "{{ .Values.integration_test.savings.contactpoint }}"
+            - name: LOAN_CONTACTPOINT
+              value: "{{ .Values.integration_test.loan.contactpoint }}"
+            - name: MOCK-SERVER_PORT
+              value: "{{ .Values.integration_test.mock-server.port }}"
+            - name: callback_url
+              value: ""
+            - name: IDENTITY-ACCOUNT-MAPPER_CONTACTPOINT
+              value: "{{ .Values.integration_test.identity-account-mapper.contactpoint }}"
+            - name: MIFOS-CONNECTOR_CONTACTPOINT
+              value: "{{ .Values.integration_test.mifos-connector.contactpoint }}"
```

Comment:
please add the extra env configuration.
it will be helpful for adding/enabling new tests without changing the ph-ee-engine chart

Reply:
Added